.env.local.default

```diff
@@ -119,7 +120,8 @@ FACEBOOK_CLIENT_ID=262344435023143
 FACEBOOK_CLIENT_SECRET=
 
 GITHUB_CALLBACK_URL=https://localhost:3030/auth/oauth/github
-GITHUB_CLIENT_ID=4b3e9e2c5fca8073809a
+GITHUB_APP_ID=156701
+GITHUB_CLIENT_ID=Iv1.2b4c5a211bf7ab05
```

Comment:
is this supposed to have changed?

Reply:
Yes. we need this information to get the GitHub app on the [service](https://github.com/XRFoundation/XREngine/pull/4423/files#diff-33f0167016721cfe6302b67b36443ff8f9850da6336a2bb2d15523791450fc5eR10).
```
const app = new App({
  appId: config.authentication.oauth.github.appid,
  privateKey: config.server.gitPem,
  oauth: {
    clientId: config.authentication.oauth.github.key,
    clientSecret: config.authentication.oauth.github.secret
  }
})
```